crates/bevy_core_pipeline/src/taa/mod.rs

```diff
@@ -130,7 +136,7 @@ pub struct TemporalAntiAliasBundle {
 /// and add the [`DepthPrepass`], [`MotionVectorPrepass`], and [`TemporalJitter`]
 /// components to your camera.
 ///
-/// Cannot be used with [`bevy_render::camera::OrthographicProjection`].
+/// Currently cannot be used with [`bevy_render::camera::OrthographicProjection`].
```

Comment:
Is there an issue tracking this that we can link here?

Reply:
https://github.com/bevyengine/bevy/issues/8423

Reply:
I'd rather not link it in code, as there's not a good spot in the documentation to put it imo...

Reply:
```suggestion
/// [Currently](https://github.com/bevyengine/bevy/issues/8423) cannot be used with [`bevy_render::camera::OrthographicProjection`].
```

What about like that? Unobtrusive, but makes it easy for maintainers or users to check if it's been fixed yet.